api-operator/pkg/interceptors/interceptors.go

```diff
@@ -34,25 +33,25 @@ const (
 var logger = log.Log.WithName("interceptors")
 
 // Handle handles ballerina and java interceptors
-func Handle(client *client.Client, instance *wso2v1alpha2.API) error {
-	// handle ballerina interceptors
-	balFound, err := handle(client, &instance.Spec.Definition.Interceptors.Ballerina, instance.Namespace, balIntPath)
-	if err != nil {
-		logger.Error(err, "Error handling Ballerina interceptors", "namespace", instance.Namespace, "apiName", instance.Name)
-		return err
-	}
-	kaniko.DocFileProp.BalInterceptorsFound = balFound
-
-	// handle java interceptors
-	javaFound, err := handle(client, &instance.Spec.Definition.Interceptors.Java, instance.Namespace, javaIntPath)
-	if err != nil {
-		logger.Error(err, "Error handling Java interceptors", "namespace", instance.Namespace, "apiName", instance.Name)
-		return err
-	}
-	kaniko.DocFileProp.JavaInterceptorsFound = javaFound
-
-	return nil
-}
+//func Handle(client *client.Client, instance *wso2v1alpha2.API) error {
+//	// handle ballerina interceptors
+//	balFound, err := handle(client, &instance.Spec.Definition.Interceptors.Ballerina, instance.Namespace, balIntPath)
+//	if err != nil {
+//		logger.Error(err, "Error handling Ballerina interceptors", "namespace", instance.Namespace, "apiName", instance.Name)
+//		return err
+//	}
+//	kaniko.DocFileProp.BalInterceptorsFound = balFound
+//
+//	// handle java interceptors
+//	javaFound, err := handle(client, &instance.Spec.Definition.Interceptors.Java, instance.Namespace, javaIntPath)
+//	if err != nil {
+//		logger.Error(err, "Error handling Java interceptors", "namespace", instance.Namespace, "apiName", instance.Name)
+//		return err
+//	}
+//	kaniko.DocFileProp.JavaInterceptorsFound = javaFound
+//
+//	return nil
+//}
 
```

Comment:
Shall we remove commented code?

Reply:
Fixed with 554ae7fb94dd54633529617b121049c3654f9f8e